install-stubs/database/migrations/fill_default_admin_user_and_permissions.php

```diff
@@ -156,7 +159,10 @@ public function up(): void
                     $roleId = $roleItem->id;
                 }
 
-                $permissionItems = DB::table('permissions')->whereIn('name', $permissions)->where(
+                $permissionItems = DB::table('permissions')
+                    ->whereIn('name', $permissions)
+                    ->where('name', '!=', 'admin.admin-user.impersonal-login')
```

Comment:
I do not understand, why is here this condition. If this is not a default permission, prepare additional array, or remove it before on line 98.